src/main/webapp/WEB-INF/tags/student/comments/feedbackSessionPanel.tag

```diff
@@ -6,8 +6,11 @@
 <div class="panel panel-primary">
     <div class="panel-heading">
         <strong>Comments in session: ${feedbackSessionRow.feedbackSessionName}</strong>
+        <div class='display-icon pull-right'>
```

Comment:
Again, there shouldn't be a need to change this tag, this tag is for feedback session panels, not comments for students panels.